app/views/ansible_variables/_fields.erb

```diff
@@ -0,0 +1,67 @@
+<div id='<%= f.object.to_param %>' class='tab-pane fields'>
+  <fieldset>
+    <h2><%= _("Ansible Variable Details") %></h2>
+    <%= text_f f, :key, :disabled => true, :size => "col-md-8" %>
+    <%= f.hidden_field :key %>
+    <%= textarea_f f, :description, :rows => :auto, :size => "col-md-8", :class => "no-stretch" %>
+    <%= select_f f, :ansible_role_id, [f.object.ansible_role], :id, :to_label, {}, :label => _("Ansible Role"), :disabled => true %>
+  </fieldset>
+  <fieldset>
+    <h2><%= _("Default Behavior") %></h2>
+    <h6><%= _("Override the default value of the Ansible variable.") %></h6>
+    <%= checkbox_f(f, :override, :onchange => 'toggleOverrideValue(this)', :size => "col-md-8",
+                   :label_help => _('Whether the Ansible variable value is managed by Foreman.')
```

Comment:
Could we describe this a bit more? Something like:

"Mark the variable to be managed by Foreman. When the Ansible role of this variable is assigned to a host, the default value will be added to Ansible inventory as a host variable. To set a different value for such variable, specify matchers."